### PR TITLE
say hello

### PR DESCRIPTION
**Pull request by bibio**
say hello

### CODE REVIEW

**Comment by bibio** [comment]
ubuntuaaaa

File: hello.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+puts "Hello!"
```
Comment (bibio) [inline hello.rb:1]: aaaab

**Review by bibio — COMMENTED** [review]
bbbb

File: hello.rb (hunk 1 above)
Comment (bibio) [inline hello.rb:1]: aaa

**Review by bibio — COMMENTED** [review]
bbbbbbc

**Review by bibio — COMMENTED** [review]
aaabc

**Review by bibio3 — APPROVED** [review]

**Pull request closed without merging**